__test__/inject.test.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { mount } from 'enzyme';
+import compose from '../compose';
+import inject from '../inject';
+
+describe('pure', () => {
+  class Title extends Component {
+    render () {
+      return (
+        <h1>{this.props.text}</h1>
```

Comment:
I think `inject` may not work in practice because IMO, blindly take property from parent node is not a good idea, for example, that prop can be a `function`, and even worse, that prop is inherited from grandparent node which very hard to debug.